### ISSUE TITLE
The network (H+) is periodically lost when the data is switched on.

### ISSUE THREAD

**Issue by IvvanVG** [z2-dev/android_device_zuk_z2_plus#114]
The network (H+) is periodically lost when the data is switched on.
- ROM build version (date): 20170904 and older, clear installation on zui 2.5.412
- Baseband version: M8996FAAAANAZM-1.70664.3.76387.1
The network is stable on baseband from android M (9-00009-M8996FAAAANAZM-1.53476.1 and other) and when the data is switched off.

[logcat.txt](https://github.com/z2plus-dev/android_device_zuk_z2_plus/files/1196561/logcat.txt)
[radio.txt](https://github.com/z2plus-dev/android_device_zuk_z2_plus/files/1196562/radio.txt)
<details>
  <summary>Screenshots</summary>
<img src="https://user-images.githubusercontent.com/8935752/28913814-e9bd5a40-786b-11e7-9195-b5a80681069b.png" alt="image alt" align="left" />
<img src="https://user-images.githubusercontent.com/8935752/28913816-e9ca335a-786b-11e7-87b1-d7d2a8fae523.png" alt="image alt" align="left" />
<img src="https://user-images.githubusercontent.com/8935752/28913815-e9c4d748-786b-11e7-8fee-21ce6d21dab3.png" alt="image alt" align="left" />
</details>

**Comment by AlvaroBrey** [comment]
> The network is stable on modems from 6 android (9-00009-M8996FAAAANAZM-1.53476.1 and other) 

I understand this modem is older than `M8996FAAAANAZM-1.70664.3.76387.1`, right? It's probably an issue of less-than-ideal compatibility between firmware and blobs.

I'll eventually make it a requirement to have the latest firmware, once I update most blobs to N.

**Comment by IvvanVG** [comment]
Right. Modem M8996FAAAANAZM-1.70664.3.76387.1 from the latest version of zui 2.5.412, modem 9-00009-M8996FAAAANAZM-1.53476.1 from zui 1.9.044.

**Comment by Luis-Gomez** [comment]
The fault occurs only in H+ but not in LTE.

Thanks

**Comment by IvvanVG** [comment]
The problem in the new build is not solved =(

**Comment by Darkiu** [comment]
In the last review, the problem still exists.

If you want, some log me and I will attach it

**Comment by IvvanVG** [comment]
The problem in the last build is not solved...

**Comment by AlvaroBrey** [comment]
Well, that means it isn't a problem with blobs-modem compatibility. For now I can only advise you to use an older modem, as this is a very time-consuming thing to investigate.

**Comment by IvvanVG** [comment]
Thanks! I checked error for zui 2.5.412 (335, 104) and the problem is the same. So this is not a bug of your firmware. I think, modems or RIL from android N have an error in the networks h+.  the modem is now installed 9-00026-M8996AAAANAZM-1.63821.1 (from zui 2.3.042, android M) - there are no problems, I think you can close the issue =]

**Comment by fernandoiepe** [comment]
until today I have problem with the network h + and with wifi .... I think I will go back with the android M, does anyone have a suggestion?

**Comment by AlvaroBrey** [comment]
@fernandoiepe As we say earlier in this issue thread, if you experience these bugs you need to flash an older baseband. No need for downgrading the whole ROM.

**Comment by fernandoiepe** [comment]
@ontherunvaro How do you do this?
